Initial ticket sales announcement (#1246)
writethedocs/www: docs/conf/portland/2020/news/online-ticket-sales.rst

:template: {{year}}/generic.html

.. post:: May 26, 2020
   :tags: portland-2020, tickets

> Tickets on sale for the Portland virtual conference

Reviewer: ```suggestion
Announcing ticket sales for the Portland virtual conference
```

===================================================

We're excited to announce that tickets are on sale for our Portland virtual conference.
More details will be coming soon,

> but we're getting excited for the accessibility of the new format,

Reviewer 2: ```suggestion
but we're enthousiastic for the accessibility of the new format,
```
Double use of excited

Reviewer 3: ```suggestion
but we're enthusiastic about the accessibility benefits of the new format,
```

and can't wait to try out a few new ideas this year.

Get your ticket for the conference
----------------------------------

**The Portland conference will be online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/portland/2020/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer.

> If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/2020/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the online event, but we might still sell out to make sure we're providing a managable event side for our attendees, as always.

Reviewer 2: event side? I don't know what that means.

> If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/2020/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the online event, but we might still sell out to make sure we're providing a managable event side for our attendees, as always.

Reviewer 2: ```suggestion
If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/2020/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the online event, but we might still sell out to make sure we're providing a managable event size for our attendees, as always.
```

We're still working out all the details for our online event,

> but know that we're going to keep as much of the spirit of the event alive:

Reviewer 2: ```suggestion
but know that we're going to keep as much as possible of the spirit of the event alive:
```

* Excellent single track of presentations -- now with moderated Q&A at the end!
* Writing Day
* Unconference sessions
* Job Fair
* Sponsor booths
* A virtual hallway track
* A social event of some kind, bring your own beverage and snacks :)
  
We really want to make sure that folks can get value from being at the event at the same time,
and we think it will be another great year of helping our community connect.

Grant program
-------------

> We have updated our grant program application to take into account the online format of the event.

Reviewer: ```suggestion
We updated our grant program application to take into account the online format of the event.
```
